
syntax = "proto2";
package fubble_rtc;

option java_package = "com.freckles.of.couple.fubble";
option java_outer_classname = "RtcMessageContainer";


// Request

message RtcMessageRequest {
	optional JoinRoom joinRoom = 1;
	optional DeleteRoom deleteRoom = 1;
}

message JoinRoom {
	optional string roomName = 1;
}

message DeleteRoom {
	optional string roomName = 1;
}





// Response

message RtcMessageResponse {
	optional RoomInformation roomInformation = 1;
}

message RoomInformation {
	optional string rtcServer = 1;
}
